coding/zip_creator.cpp

```diff
@@ -67,59 +51,20 @@ int GetCompressionLevel(CompressionLevel compression)
 }
 }  // namespace
 
-bool CreateZipFromPathDeflatedAndDefaultCompression(std::string const & filePath,
-                                                    std::string const & zipFilePath)
+void FillZipLocalDateTime(zip::DateTime & res)
 {
-  // Open zip file for writing.
-  SCOPE_GUARD(outFileGuard, [&zipFilePath]() { base::DeleteFileX(zipFilePath); });
-
-  ZipHandle zip(zipFilePath);
-  if (!zip.Handle())
-    return false;
-
-  zip::FileInfo zipInfo = {};
-  CreateTMZip(zipInfo.tmz_date);
-
-  std::string fileName = base::FileNameFromFullPath(filePath);
-  if (!strings::IsASCIIString(fileName))
-    fileName = "OrganicMaps.kml";
-
-  if (zip::Code::Ok != zip::OpenNewFileInZip(zip.Handle(), fileName, zipInfo, "ZIP from OMaps",
-                                             Z_DEFLATED, Z_DEFAULT_COMPRESSION))
-  {
-    return false;
-  }
-
-  // Write source file into zip file.
-  try
-  {
-    base::FileData file(filePath, base::FileData::OP_READ);
-    uint64_t const fileSize = file.Size();
-
-    uint64_t currSize = 0;
-    std::array<char, zip::kFileBufferSize> buffer;
-    while (currSize < fileSize)
-    {
-      auto const toRead = std::min(buffer.size(), static_cast<size_t>(fileSize - currSize));
-      file.Read(currSize, buffer.data(), toRead);
-
-      if (zip::Code::Ok != zip::WriteInFileInZip(zip.Handle(), buffer, toRead))
-        return false;
-
-      currSize += toRead;
-    }
-  }
-  catch (Reader::Exception const & ex)
-  {
-    LOG(LERROR, ("Error reading file:", filePath, ex.Msg()));
-    return false;
-  }
-
-  outFileGuard.release();
-  return true;
+  auto const now = std::time(nullptr);
+  // Files inside .zip are using local date time format.
+  auto const * local = std::localtime(&now);
+  res.tm_sec = local->tm_sec;
+  res.tm_min = local->tm_min;
+  res.tm_hour = local->tm_hour;
+  res.tm_mday = local->tm_mday;
+  res.tm_mon = local->tm_mon;
+  res.tm_year = local->tm_year;
 }
 
-bool CreateZipFromFiles(std::vector<std::string> const & files, std::string const & zipFilePath,
+bool CreateZipFromFiles(std::vector<std::string> const & files, std::vector<std::string> const & filesInArchive, std::string const & zipFilePath,
```

Comment:
The first line should be an ASSERT_EQUAL that compares sizes of both vectors.

Reply:
Fixed